### PR TITLE
Add commwebapp config

### PR DESCRIPTION
**Pull request by ErlendLygre**

### CODE REVIEW

File: src/Equinor.ProCoSys.Config/CommWebApp/Configuration.cs [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+            {
+                return new BadRequestObjectResult("Invalid origin");
+            }
+            var environment = EnvironmentSelector.GetEnvironment(originHeader.Value);
+            if (string.IsNullOrWhiteSpace(environment))
+            {
+                return new BadRequestObjectResult("Invalid origin");
+            }
+
+            var configConnectionString = Environment.GetEnvironmentVariable("FrontendConfig");
```
Comment (Cnordbo) [inline src/Equinor.ProCoSys.Config/CommWebApp/Configuration.cs:60]: This needs to be setup using a new CommWebAppConfig with a new App Configuration instance

File: src/Equinor.ProCoSys.Config/CommWebApp/Auth.cs [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    public static class Auth
+    {
+        [FunctionName("CommWebAppAuth")]
+        public static IActionResult Run(
+            [HttpTrigger(AuthorizationLevel.Anonymous, "get", Route = "CommWebApp/Auth")] HttpRequest req,
+            ILogger log)
+        {
+            log.LogInformation("Processing configuration request...");
+
+            var configConnectionString = Environment.GetEnvironmentVariable("FrontendConfig");
```
Comment (Cnordbo) [inline src/Equinor.ProCoSys.Config/CommWebApp/Auth.cs:24]: This needs to be setup using a new CommWebAppConfig with a new App Configuration instance

**Review by Cnordbo — CHANGES_REQUESTED** [review]
We need to swap out the Frontend configuration instance with a new instance of App Configuration tailored for the webapps. This is done because the applications will use different ClientIds, scopes and feature flag granulation. 

Remember to talk to DevOps about setting this up in ARM after initial setup.

File: src/Equinor.ProCoSys.Config/CommWebApp/Configuration.cs (hunk 1 above)
Comment (ErlendLygre) [reply src/Equinor.ProCoSys.Config/CommWebApp/Configuration.cs]: This is now updated from "FrontendConfig" to "CommWebAppConfig". CommWebAppConfig has been added as a key in pcs-config-non-prod-func > Configuration, with the value being the endpoint to a separate app config instance for the Commissioning webapp.

File: src/Equinor.ProCoSys.Config/CommWebApp/Auth.cs (hunk 2 above)
Comment (ErlendLygre) [reply src/Equinor.ProCoSys.Config/CommWebApp/Auth.cs]: This is now updated from "FrontendConfig" to "CommWebAppConfig". CommWebAppConfig has been added as a key in pcs-config-non-prod-func > Configuration, with the value being the endpoint to a separate app config instance for the Commissioning webapp.

**Review by Cnordbo — APPROVED** [review]

**Pull request merged**